### PR TITLE
[#73] Add documenation for the documentation styleguide.

### LINKED ISSUE #73: Agree on a style for documentation writing and apply it
This is not covered in geck but it should be covered for future reference.

The following are just proposals based on how we already write / review documentation.

Once we agree, create a ticket to check and apply for future reviews.

**Language:** American English (ie `e.g. using a Domain Controller or other centralized identity
management systems`)

**How units are written:** SI style (English version) or International System of Units
For example, if writing large numbers use the example at  https://en.wikipedia.org/wiki/Decimal_mark#Examples_of_use (decimals) ie `100 000`.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

Fixes #73 

Changes
=======

Add an initial page.

As a drive-by the theme was updated to Furo so that we have dart and light version.

Also, the project was updated to Python 3 and latest sphinx version.

How to try and test the changes
===============================

reviewers: @dumol @MohamedWasimMohamedAkram 

check that changes make sense

Documentation preview here https://chevah-geck--89.org.readthedocs.build/styleguide/documentation.html

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

File: README.rst [hunk 1]
```diff
@@ -16,4 +18,4 @@ The pages are managed using `Sphinx Docs <https://www.sphinx-doc.org>`_.
 To generate a local copy::
 
   ./brink.sh deps
```
Comment (adiroiban) [inline README.rst:20]:
```suggestion
  ./pythia.sh deps
```

File: README.rst [hunk 2]
```diff
@@ -16,4 +18,4 @@ The pages are managed using `Sphinx Docs <https://www.sphinx-doc.org>`_.
...
-  ./brink.sh generate
+  ./brink.sh build
```
Comment (adiroiban) [inline README.rst:21]:
```suggestion
  ./pythia.sh build
```

**Comment by dumol** [comment]
Some minor improvements in my commit above. Please double-check, thanks!

Also, couldn't run `./pythia.sh deps` on my musl-based workstation:
```
INFO: pip is looking at multiple versions of furo to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement sphinx-basic-ng>=1.0.0.beta2 (from furo) (from versions: none)
ERROR: No matching distribution found for sphinx-basic-ng>=1.0.0.beta2
```

**Comment by adiroiban** [comment]
Thansk... not sure what is going on there

It looks like wants to install `sphinx-basic-ng`  ... but this is missing for our pypi.

I have copied it.

BUt on my system, the deps for furo are:

```
Requirement already satisfied: Sphinx==7.3.7 in ./build/lib/python3.11/site-packages (from -r docs/requirements.txt (line 1)) (7.3.7)
Requirement already satisfied: furo in ./build/lib/python3.11/site-packages (from -r docs/requirements.txt (line 2)) (2024.5.6)
Requirement already satisfied: sphinxcontrib-applehelp in ./build/lib/python3.11/site-packages (from Sphinx==7.3.7->-r docs/requirements.txt (line 1)) (1.0.4)
Requirement already satisfied: sphinxcontrib-devhelp in ./build/lib/python3.11/site-packages (from Sphinx==7.3.7->-r docs/requirements.txt (line 1)) (1.0.2)
Requirement already satisfied: sphinxcontrib-jsmath in ./build/lib/python3.11/site-packages (from Sphinx==7.3.7->-r docs/requirements.txt (line 1)) (1.0.1)
Requirement already satisfied: sphinxcontrib-htmlhelp>=2.0.0 in ./build/lib/python3.11/site-packages (from Sphinx==7.3.7->-r docs/requirements.txt (line 1)) (2.0.1)
Requirement already satisfied: sphinxcontrib-serializinghtml>=1.1.9 in ./build/lib/python3.11/site-packages (from Sphinx==7.3.7->-r docs/requirements.txt (line 1)) (1.1.10)
Requirement already satisfied: sphinxcontrib-qthelp in ./build/lib/python3.11/site-packages (from Sphinx==7.3.7->-r docs/requirements.txt (line 1)) (1.0.3)
R
```

**Comment by dumol** [comment]
Thanks! I have `deps` working now on my system.

**changes-approved**

**Review by dumol — APPROVED** [review]

**Pull request merged**